contrib/vexecutor/vadt.c

```diff
@@ -39,10 +39,11 @@ size_t v##type##Size(vheader *vh) \
 vheader* buildv##type(int n) \
 { \
 	vheader* result; \
-	result = (vheader*) palloc0(offsetof(v##type,values) + n * sizeof(type)); \
+	result = (vheader*) malloc(offsetof(v##type,values) + n * sizeof(type)); \
+	memset(result,0,offsetof(v##type,values) + n * sizeof(type)); \
 	result->dim = n; \
 	result->elemtype = typeoid; \
-	result->isnull = palloc(sizeof(bool) * n); \
+	result->isnull = malloc(sizeof(bool) * n); \
```

Comment:
why use malloc here? and the below code is still palloc()

Reply:
It does make sense, I will attach another commit change it